myfitnesspal/client.py

```diff
@@ -57,7 +57,10 @@ def _login(self):
                 'password': self.password,
             }
         )
-        if 'Incorrect username or password' in result.content:
+        # result.content is bytes so we decode it ASSUMING utf8 (which may be a
+        # bad assumption?) PORTING_CHECK
```

Comment:
Yeah -- that seems like a reasonable assumption; I just glanced at the content from MFP, and it does suggest that it's UTF-8 --

```
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```